aws/resource_aws_ecs_task_definition.go

```diff
@@ -230,6 +230,70 @@ func resourceAwsEcsTaskDefinition() *schema.Resource {
 				}, false),
 			},
 
+			"proxy_configuration": {
+				Type:     schema.TypeSet,
```

Comment:
I think this should be `schema.TypeList` followed by `MaxItems: 1` per the [contributing] guidelines.

[contributing]: https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md

Reply:
Thanks for the feedback. Addressed with 26e6797b3dde798a8082df1360aab0eebcc227e1